azure/services/tags/mock_tags/doc.go

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 //
 //go:generate ../../../../hack/tools/bin/mockgen -destination client_mock.go -package mock_tags -source ../client.go Client
 //go:generate ../../../../hack/tools/bin/mockgen -destination tags_mock.go -package mock_tags -source ../tags.go TagScope
+//go:generate ../../../../hack/tools/bin/mockgen -destination ta -package mock_tags -source ../tags.go TagsClient
```

Comment:
> ta

is this a typo?

Reply:
This is indeed a typo.  I'm not sure how it got generated

Reply:
Removing and resolving.